src/modules/apps/e2e/index.ts

```diff
@@ -0,0 +1,130 @@
+import chalk from 'chalk'
```

Comment:
If we test this plugin without being on toolbelt this dependency will not be resolved, since it's not on package.json's dependencies